packages/client/modules/meeting/helpers/ensureHasText.ts

```diff
@@ -0,0 +1,2 @@
+const ensureHasText = (value: string) => value.trim().length
```

Comment:
I think we could define this function in the same file as the function is very small, a little bit of redundancy is fine I guess.